demo/components/debug-demo.js

```diff
@@ -0,0 +1,66 @@
+/*eslint-disable no-magic-numbers */
```

Comment:
A demo to work through bug reproductions. Intensionally not shown on the demo table of contents